### DISCUSSION TITLE
Ubuntu 21.2 Segmentation fault

### DISCUSSION THREAD

**Discussion by Glitch-Artist** [obsproject/obs-studio#10003] [Development Questions]

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
